bot.js

```diff
@@ -85,12 +86,21 @@ class EchoBot {
                 count = count === undefined ? 1 : ++count;
 
                 const results = await this.luisRecognizer.recognize(turnContext);
-                console.log('JSON(results): ', JSON.stringify(results));
+                // console.log('JSON(results): ', JSON.stringify(results));
+                // console.dir(results);
                 const topIntent = LuisRecognizer.topIntent(results);
                 const confidence = results.luisResult.topScoringIntent.score;
-                const entity = results.luisResult.entities[0]['entity'];
-                await turnContext.sendActivity(`Dunno, but I'm ${Math.floor(confidence * 100)}% sure your intent is ${ topIntent }, and the thing you're asking about is ${ entity }!`);
+
+                // const entity = results.luisResult.entities[0]['entity'];
+                //await turnContext.sendActivity(`Dunno, but I'm ${Math.floor(confidence * 100)}% sure your intent is ${ topIntent }, and the thing you're asking about is ${ entity }!`);
+
+                const appName = results.entities.appName;
+                const releaseData = await this.retrieveLatestGithubRelease(appName);
+                // console.dir(releaseData);
+                await turnContext.sendActivity(`The latest release for ${appName} is '${releaseData.name}', created by ${releaseData.author.login} on ${releaseData.created_at}`);
```

Comment:
good stuff. will need to add some control statements to identify which intent it is and handle it specifically though

Reply:
i imagine that would make another great PR!